content/blog/crossing-realities-ready-to-daydream.md

```diff
@@ -0,0 +1,117 @@
+
+
+## How do I start?
+
+If you want to have fun with the controller and create your own Daydream apps in [Unity](https://unity3d.com/ "Unity") here what worked for me:
+
+### 1. Get Daydream technical preview
+You need a special version of Unity called *Unity Daydream technical preview*. You can get it [*here*](https://unity3d.com/partners/google/daydream "Daydream technical preview"). Once you've downloaded it install it like a normal Unity version, it should be rather straightforward but if you have difficulties look at my previous blog post on how to [*configure Unity for Cardboard*](/blog/crossing-realities-the-beginning-of-a-VR-journey/ "The beginning of a VR journey"). By default it should install next to your normal Unity version (if you have any) and not overwrite it. You can check the name of the installation folder to check this. it also means that you must be careful and not open the wrong version of Unity...
+
+While it's installing you can download Android SDK 24.
+
+### 2. Download Android Nougat SDK
+In order to compile your Daydream apps you need the *SDK Platform* and *Android SDK Build-tools* for *Android 7.0 (API 24)*. To get that, launch the *Android SDK Manager*, either from Android Studio as described [here](https://developer.android.com/studio/intro/update.html#sdk-manager "Android SDK Manager") or from command line: Mac `~/Library/Android/sdk/tools/android` and Windows `C:\Users\<user_name>\AppData\Local\Android\Sdk\tools\android.bat` (replace `<user_name>` by your actual user name). WARNING: Those paths may vary depending on how you installed the SDK Manager.
+
+If you don't have *Android SDK Manager* then you can either download all *Android Studio* or *Get just the command line tools* [here](https://developer.android.com/studio/index.html#downloads "Android Studio"). If you choose the second option extract the file to the desired location and launch `./tool/android` from the command line.
+
+Then you should have a window like this:
+
+![article-img-centered](/img/blog/0009/android_sdk_manager.PNG "Android SDK Manager")
+
+Check the boxes as shown in the image. The *Android SDK build-tools* 24.0.1, 24.0.2 and 24.0.3 may not be needed but I installed them just in case.
+Then click on *Install packages...* and go get a coffee while everything is installing.  
+
+### 3. Set the Android SDK path in Unity
+Now you need to set the Android SDK location in Unity (the new *Daydream technical preview* Unity version that you just installed).
+To do this open Unity and create a new empty project. Go to the preference menu (Mac `Unity > Preferences...`, Windows `Edit > Preferences...`) and to the `External Tools` tab (on the left).
+Here you can check the Android SDK path, make sure it's the same that the one indicated at the top of the *Android SDK Manager* window.
+
```

Comment:
make sure it's the same that the one indicated => make sure it's the same AS the one indicated